CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 ## Bug Fixes
 
 * Assigning a managed object's own list to itself would accidentally clear it (#5395).
+* Don't try to acquire `ApplicationContext` if not available in `Realm.init(Context)` (#5389).
```

Comment:
this was already merged to `master` ?

Reply:
Yes

Reply:
But the changelog was slightly misleading. So I'm just fixing it here. Sorry 😄